site/realm/data_sources/mongodb-atlas/aiidprod/notifications/rules.json

```diff
@@ -0,0 +1,37 @@
+{
+    "collection": "notifications",
+    "database": "aiidprod",
```

Comment:
I would move this to the `customData` database because it is not related to the actual "data" of the database. Also, it might get us in trouble if we do a restore of the `aiidprod` database.

Reply:
Done! `notifications` collection moved to `customData` DB